src/sdk/formatting/formatDate.ts

```diff
@@ -99,6 +106,16 @@ const relativeTimeLabels = {
 	dd: '%d days ago'
 };
 
+const futureRelativeTimeLabels = {
```

Comment:
These do not follow our [data formatting guidelines](https://skysync.atlassian.net/wiki/spaces/PUCS/pages/323649654/Data+formats).

Reply:
Would I only use Today, Tomorrow, or a date instead of these?

Reply:
The guidelines only call out distant past, not distant future.